dduper

```diff
@@ -23,7 +23,7 @@ import sys
 from collections import OrderedDict
 from fcntl import ioctl
 from itertools import combinations
-from itertools import izip_longest
+from itertools import zip_longest
```

Comment:
Hello @Lakshmipathi, I looked into the issue. It seems that the problem is with 'float' if I cast **n** to 'int' it works perfectly fine. As far as zip_longest is concerned, it is just the new library in python3.x instead of izip_longest. Does it matter value of **n** to be float or int?

Reply:
`n` needs to int. Please cast that value to `int`. I believe that should work.

Reply:
It's done. Please check on your side and let me know. Thank you.

Reply:
thanks, will test and let you the results :+1: